docs/content/integrations/airflow.mdx

```diff
@@ -20,22 +20,25 @@ This integration is designed to help support users who have existing Airflow usa
+| Sensor                | [Sensor](/concepts/partitions-schedules-sensors/sensors)                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                       |
+| DAG run               | Job run                                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                       |
+| Plugins/Providers     | [Integrations](/integrations)                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                       |
+| Datasets              | [Software-defined assets (SDAs)](/concepts/assets/software-defined-assets)                                                                                                                                                                                                                                                                        | SDAs are more powerful and mature than datasets and include support for things like partitioning.                                                                                                     |
+| Connections/Variables | [Run config](/concepts/configuration/config-schema), [Configured API](/concepts/configuration/configured) and [environment variables](https://docs.dagster.io/dagster-cloud/developing-testing/environment-variables-and-secrets#dagster-cloud-environment-variables-and-secrets) (Dagster Cloud only)                                            |                                                                                                                                                                                                       |
```

Comment:
```suggestion
| Connections/Variables | [Run config](/concepts/configuration/config-schema), [Configured API](/concepts/configuration/configured) and [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets#dagster-cloud-environment-variables-and-secrets) (Dagster Cloud only)                                            |                                                                                                                                                                                                       |
```

Reply:
I tried making this change, but the build is claiming that the link is dead after I do that. So maybe there's a reason this was put there before? This was already here before the change, proposed in this PR, so I think best to leave it be here.